### ISSUE TITLE
Clear destination directory before copy

### ISSUE THREAD

**Issue by artyom-p** [varunsridharan/action-github-workflow-sync#18]
It would be really nice to have the ability to clean up the destination directory before copying files.

**Comment by varunsridharan** [comment]
Thanks for the suggestion. soon i will add it.

**Comment by chburmeister** [comment]
is this repo still maintained? We would love to contribute similar stuff :-)

**Comment by varunsridharan** [comment]
@chburmeister yes this repository is active and is currently lacks support because i am stuck with some personal works soon i will become free and will be able to work on this near feature. you are welcome to send us pull request of the features you want and it will be mereged after reviews asap.
